utils/build-script-utils/src/git.rs

```diff
@@ -34,15 +34,15 @@ pub fn rerun_if_git_head_changed() {
 				eprintln!("cargo:warning=Unable to read the Git repository: {}", err);
 
 				return;
-			}
-			Ok(None) => {}
+			},
```

Comment:
They're added because of the `match_block_trailing_comma = true` directive. I'd personally prefer to not set that directive, but I don't feel very strongly either way.

Reply:
Removed this.